README.md

```diff
@@ -0,0 +1,19 @@
+Hello-Microverse
+===============
+Languages used: HtML5 and CSS3.
```

Comment:
- [ ] **Required:** Please use the Microverse template [README](https://github.com/microverseinc/readme-template) to built a descriptive README file.